apps/examples/testcase/le_tc/network/itc_net_setsockopt.c

```diff
@@ -0,0 +1,234 @@
+/****************************************************************************
+*
+* Copyright 2017 Samsung Electronics All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+* either express or implied. See the License for the specific
+* language governing permissions and limitations under the License.
+*
+****************************************************************************/
+
+/// @file itc_net_setsockopt.c
+/// @brief ITC Test Case Example for setsockopt() API
+
+#include <sys/types.h>
+#include <sys/socket.h>
+
+#include "tc_internal.h"
+
+/**
+* @fn                   :itc_net_setsockopt_n_invalid_opt_name
+* @brief                :setsockopt accept opt name
+* @Scenario             :sending invalid opt name
+* @API's covered        :setsockopt()
+* Preconditions         :socket file descriptor
+* Postconditions        :None
+*/
+static void itc_net_setsockopt_n_invalid_opt_name(int s)
+{
+	int ret = -1;
+
+	ret = setsockopt(s, SOL_SOCKET, 16, 0, 0);
```

Comment:
Could you tell me what do the paramesters 16, 0 and 0 mean?
It would be helpful to write comments about the parameters

Reply:
Fixed review comments. Used valid variable names for these parameters